docs/support.md

```diff
@@ -0,0 +1,52 @@
+# Support
+
+The following features are supported by envd team.
+
+> envd v0.2.5
+
+## OS support
+
+- ubuntu 20.04
```

Comment:
We are able to support any Debian-based image now (with `# syntax=v1`)

Reply:
marked and I will have a try next week.